jupyterlab_bigquery/src/index.ts

```diff
@@ -6,17 +6,25 @@ import {
   JupyterFrontEndPlugin,
 } from '@jupyterlab/application';
 
-import { ListItemsWidget } from './components/list_items_panel/list_tree_item_widget';
+import ListItemsWidget from './components/list_items_panel/list_tree_item_widget';
 import { ListProjectsService } from './components/list_items_panel/service/list_items';
 import { WidgetManager } from './utils/widget_manager';
+import { ReduxReactWidget } from './utils/ReduxReactWidget';
 
 async function activate(app: JupyterFrontEnd) {
-  const manager = new WidgetManager(app);
+  const manager = WidgetManager.getInstance(app);
   const context = { app: app, manager: manager };
   const listProjectsService = new ListProjectsService();
-  const listWidget = new ListItemsWidget(listProjectsService, context);
-  listWidget.addClass('jp-BigQueryIcon');
-  app.shell.add(listWidget, 'left', { rank: 100 });
+  WidgetManager.getInstance(app).launchWidget(
```

Comment:
minor nit: but it might be more clear to use `manager.launchWidget(...` since it should be the same variable.